src/Game/Command/WeatherCommand.php

```diff
@@ -27,7 +27,22 @@ public function fire()
             return;
         }
 
-          $this->gameManager->sendMessageToChannel($this->game, ":rain_cloud: It is raining. It is a cold rain, and the freezing drops chill you to the bone." );
-
+        $weather= $this->game->weather;
```

Comment:
Okay. Do I need to make another PR or is there a way to modify the current one ?